components/Pagination.js

```diff
@@ -0,0 +1,90 @@
+import styled from 'styled-components';
+
+const PaginationContainer = styled.div`
+  text-align: center;
+`;
+
+const BasePaginationButton = styled.button`
+  ${(p) =>
+    p.contentUrl &&
+    `&:before {
+        display: inline-block;
+        height: 24px;
+        line-height: normal;
+        content: url(${p.contentUrl});
+    }`}
+  font-family: HK Grotesk;
```

Comment:
NIt: do we need this if we defined it as our global font?

Reply:
Thanks for the speedy feedback!
I tried not specifying the `font-family`, but for some reason, the font for the `button` component defaulted to Arial, so I had to specify HK Grotesk
<img width="577" alt="Screen Shot 2021-08-20 at 1 06 51 PM" src="https://user-images.githubusercontent.com/38872354/130288053-cfc5104a-ee6b-441c-90f3-f4815d4afaed.png">

Reply:
Oh that's weird, thanks for the response 🚀 Lgtm!